assets/src/animation/effects/backgroundPan/index.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { BACKGROUND_ANIMATION_EFFECTS, DIRECTION } from '../../constants';
+import SimpleAnimation from '../../parts/simpleAnimation';
+import { getMediaBoundOffsets } from '../../utils';
+
+export function EffectBackgroundPan({
+  panDir = DIRECTION.RIGHT_TO_LEFT,
+  duration = 500,
+  delay,
+  easing,
+  element,
+}) {
+  const timings = {
+    fill: 'both',
+    duration,
+    delay,
+    easing,
+  };
+
+  const animationName = `direction-${panDir}-${BACKGROUND_ANIMATION_EFFECTS.PAN.value}`;
+
+  const offsets = getMediaBoundOffsets({ element });
+  const translateToOriginX = 'translate3d(0%, 0, 0)';
+  const translateToOriginY = 'translate3d(0, 0%, 0)';
```

Comment:
just curious! Is there a reason for using `0%` vs `0` here?

Reply:
yea the web animation api is just super picky and weird. it needed `0` on the dimensions that weren't transitioning, & `0%` on the dimensions that were transitioning between percents, otherwise it complained about invalid keyframes

Reply:
ah, that makes sense - i wasn't thinking about the transitions. cool!